aten/src/ATen/native/mkldnn/xpu/qlinear.cpp

```diff
@@ -0,0 +1,135 @@
+#include <torch/library.h>
+
+#include <ATen/native/mkldnn/xpu/detail/oneDNN.h>
+#include <c10/core/ScalarType.h>
+
+using namespace at::native::onednn;
+
+namespace at::native::xpu {
+
+Tensor q_linear_pointwise(
+    Tensor act, // int8 cpu tensor
+    double act_scale,
+    int64_t act_zero_point,
+    Tensor weight,
+    Tensor weight_scales,
+    Tensor weight_zero_points,
+    c10::optional<Tensor> bias,
```

Comment:
```suggestion
    std::optional<Tensor> bias,
```

Reply:
Thanks for you kindly reminding. The code has been changed.